exercise_08/formatStringByWordsLength.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Write JS function "formatStringByWordsLength" to check number of words contained in given string
+    - if total number of words is greater then 5, then return uppercase string
+    - otherwise, return lower case string
+ * input: string (sentence) 
+ * output: string
+ */
+
+/**
+ * CASE #1
+ * input: "This is a long, long, long sentence."
+ * output: "THIS IS A LONG, LONG, LONG SENTENCE."
+ */
+
+/**
+ * CASE #2 * input: "This is short one."
+ * output: "this is short one."
+ */
+import getTotalWordsFromString from "../exercise_02/getTotalWordsFromString.js";
+import isGreaterThan from "../exercise_04/isGreaterThan.js";
+
+/**
+ *  Check if the given sentence is longer than the given number of words and change to lower case or upper case string
+ * @param {string} sentence Given sentence to evaluate
+ * @returns {string} Return the sentence in LowerCase or UpperCase
+ */
+const formatStringByWordsLength = function (sentence) {
+  if (isGreaterThan(5, getTotalWordsFromString(sentence))) {
```

Comment:
One line, one thing.
```suggestion
  const words =  getTotalWordsFromString(sentence);
  const exceeds = isGreaterThan(words, 5)
  if (exceeds) {
```

FIX: needs work